examples/AnimatedGIF/ASAnimatedImage/ViewController.m

```diff
@@ -32,7 +30,9 @@ - (void)viewDidLoad {
   // Do any additional setup after loading the view, typically from a nib.
   
   ASNetworkImageNode *imageNode = [[ASNetworkImageNode alloc] init];
-  imageNode.URL = [NSURL URLWithString:@"https://s-media-cache-ak0.pinimg.com/originals/07/44/38/074438e7c75034df2dcf37ba1057803e.gif"];
+//    imageNode.URL = [NSURL URLWithString:@"https://s-media-cache-ak0.pinimg.com/originals/07/44/38/074438e7c75034df2dcf37ba1057803e.gif"];
+//  imageNode.URL = [NSURL URLWithString:@"https://storage.googleapis.com/downloads.webmproject.org/webp/images/dancing_banana2.lossless.webp"];
```

Comment:
Let's remove these lines.

Reply:
I think I'll remove one and add a comment to the other one saying "uncomment to see animated webp support"?